notebook/supermusr-plotting-frames.ipynb

```diff
@@ -0,0 +1,127 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 40,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# These values are set here and used further down\n",
+    "filename = \"trace-Tue 26 Jul 10-57-38 BST 2022.h5\"\n",
+    "# plot_frame = True\n",
+    "frame = 400 # Expected if plot_frame is True, alternatively use a timestamp\n",
+    "timestamp = 1658826552.89 # Expected in seconds UTC (Note: the data takes into consideration nanoseconds) Example: 1658826552.89\n",
+    "detector_to_plot = 0 # The value of the detector to plot"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 41,
+   "outputs": [
+    {
+     "data": {
+      "text/plain": "400"
+     },
+     "execution_count": 41,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import h5py\n",
+    "\n",
+    "data = h5py.File(filename, 'r')\n",
+    "detector_data = data['detector_data']\n",
+    "seconds_list = data['frame_timestamp']['seconds']\n",
+    "nanoseconds_list = data['frame_timestamp']['nanoseconds']\n",
+    "frame_list = data['frame_start_index']\n",
+    "\n",
+    "# Ensure that the frame is set, and if not then try find it from the timestamp\n",
+    "if frame is None:\n",
+    "    if timestamp is None:\n",
+    "        raise Exception(\"If frame is None, timestamp needs to be given for the frame otherwise we can't determine the frame to output\")\n",
+    "    else:\n",
+    "        # Work out the frame from the timestamp\n",
+    "        for index in range(0, len(seconds_list)-1):\n",
+    "            first = seconds_list[index] + nanoseconds_list[index] * 1e-9\n",
+    "            second = seconds_list[index+1] + nanoseconds_list[index+1] * 1e-9\n",
+    "            if first < timestamp <= second:\n",
+    "                frame = index\n",
+    "                break\n",
+    "\n",
+    "if frame is None:\n",
+    "    raise Exception(\"Frame was not set, and the frame could not be found from input data\")\n",
+    "frame"
```

Comment:
It is worth keeping this output, but add a more descriptive message as to what this is.